membrane_curvature/surface.py

```diff
@@ -87,6 +87,10 @@ def get_z_surface(coordinates, n_x_bins=10, n_y_bins=10, x_range=(0, 100), y_ran
     for l, m, z in zip(cell_x_floor, cell_y_floor, z_coords):
 
         try:
+            if l < 0 or m < 0:
```

Comment:
without this condition, negative indexes will pass regardless because Python returns the i-th element `(-1, 0) = (i, 0)`. So previously, all the negative indexes were in fact passed.

Reply:
If you warn you can also add a test that it happens!